thaomonster/censor-keywords
# Feature/input functionality

### What does this PR do? 
>Clearly and concisely describe the feature. 
This PR adds very basic functionality to access keyword inputs as well as button is able to function with a `console.log`.

### Is this a feature or Fix/Refactor?  
Feature 

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.
None

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
`src/app/app.component.ts` lines: 5-26, 31-34;

### How should this be tested?
>What steps need to be taken to confirm functionality? 
1. `git fetch` this branch
2. In terminal run `ng serve` to open the application
3. Open dev tools and click on the submit button, in the console you should see working.

### Steps To Take
>Any questions and possible steps moving forward.
Need to add the necessary logic for full functionality.

### This PR is related to which issues:
>Tag issue here: `Closes #<issue_number>`
Closes #7
